README.md

```diff
@@ -14,41 +13,30 @@ The following digest/hashing algorithms are supported:
 
 ### Digest on a single input
 
-Use the `sha256` property to obtain an instance of the `Sha256` class.
-Or use the `sha1` or `md5` properties to use those other digest
-algorithms.
+Use `sha256.convert()`, `sha1.convert()`, or `md5.convert()` to hash a
+list of bytes.
 
-Invoke the `convert` method, passing the input data as a sequence of
-bytes (i.e. `List<int>`). It returns a `Digest` object, where the
-digest value can be obtained as bytes in a `List<int>` or encoded in
-hexadecimal as a `String`.
+These methods return a [`Digest`][Digest] object, where the hash value
+can be obtained as a list of bytes or encoded in hexadecimal.
+
+The variables [`sha256`][sha256], [`sha1`][sha1], and [`md5`][md5] are
+instances of the [`Sha256`][Sha256], [`Sha1`][Sha1], and [`MD5`][MD5]
+classes, respectively.
```

Comment:
I don't think this sentence is particularly useful. It's easy for users to look at the documentation and see the class structure, but in terms of real-world usage they're going to be dealing exclusively with the constants themselves.